lib/Controller/Display.php

```diff
@@ -178,14 +185,86 @@ function displayPage(Request $request, Response $response)
 
         // Call to render the template
         $this->getState()->template = 'display-page';
+
+        $mapConfig = [
+        'setArea' => [
+            'lat' => 51.505,
+            'long' => -0.09,
+            'zoom' => 13
+        ]
+    ];
+
         $this->getState()->setData([
+            'mapConfig' => $mapConfig,
             'displayGroups' => $this->displayGroupFactory->query(),
             'displayProfiles' => $displayProfiles
         ]);
 
         return $this->render($request, $response);
     }
 
+    /**
+     * Display in map
+     * @param Request $request
+     * @param Response $response
+     * @return \Psr\Http\Message\ResponseInterface|Response
+     * @throws GeneralException
+     * @throws NotFoundException
+     * @throws \Xibo\Support\Exception\ControllerNotImplemented
+     */
+    function displayMap(Request $request, Response $response)
+    {
+        $parsedQueryParams = $this->getSanitizer($request->getQueryParams());
+
+        $filter = [
```

Comment:
Is this filter common between this method and the existing `grid` one? If it is, could we abstract it?

Reply:
Abstracted in a public function.